game/scripts/npc/abilities/alchemist_chemical_rage.txt

```diff
@@ -23,7 +23,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "55.0"
+    "AbilityCooldown"                                     "55.0 50.0 45.0 40.0 35.0" //OAA
```

Comment:
I am against this change. It could create problems in the future. Alchemist needs small dmg buffs because the hero is already good, just not popular. Alchemist will become better when we fix the bounty runes not giving bonus gold to him.